src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
oh, and btw, the default is not `null`, it's `OrleansConfigurationForTesting.xml` (and the client version of it), so the reason not to just avoid `StandardLoad()` is because they had set that property to null explicitly in the past (to get the StandardLoad behavior).